scripts/InstallHootDevelopmentEnvironment.sh

```diff
@@ -54,10 +54,27 @@ if ! dpkg -l | grep --quiet wamerican-insane; then
     sudo dpkg-reconfigure dictionaries-common
 fi
 
-# If you're fed up with unity, then uncomment this for xfce; use 'xfce4-display-settings -m' for display config
+sudo apt-get autoremove -y
+
+# If you're fed up with unity, then uncomment this for xfce; use 'xfce4-display-settings -m' after install for display config
 # sudo apt-get install xfce4
 
-# Install Chrome browser for Cucumber
+if ! grep --quiet "export HOOT_HOME" ~/.profile; then
+    echo "Adding hoot home to profile"
+    echo "export HOOT_HOME=/home/vagrant/hoot" >> ~/.profile
+fi
+if ! grep --quiet "\$HOME/.gem/ruby/1.9.1/bin:\$HOME/bin:\$HOOT_HOME/bin" ~/.profile; then
+    echo "Adding path vars to profile"
+    echo "export PATH=\$PATH:\$HOME/.gem/ruby/1.9.1/bin:\$HOME/bin:\$HOOT_HOME/bin" >> ~/.profile
+fi
+source ~/.profile
+
+#sudo gem install mime-types -v 2.6.2
+#sudo gem install capybara -v 2.5.0
+sudo gem install --user-install selenium-cucumber capybara capybara-webkit rspec
+#sudo gem install --user-install selenium-cucumber capybara-webkit rspec
+#sudo gem install cucumber capybara-webkit selenium-webdriver rspec capybara-screenshot
```

Comment:
Oh, I guess I inadvertently dropped some of the gems Brian had initally added.  I was having ruby problems early on, and I think I synced these steps up with what was in the original cucumber install instructions you wrote as a sanity check to get things working.  But Brian must have added the extra gems for some reason, so I'll add them back in.